extra/message_hexdump.go

```diff
@@ -0,0 +1,110 @@
+package extra
```

Comment:
Package should be `testutil` . Please move this file to the corresponding location .